src/StoreKeeper/WooCommerce/B2C/Frontend/Handlers/AddressFormHandler.php

```diff
@@ -168,9 +172,8 @@ private function addFields(array $fields)
                 'type' => 'text',
                 'priority' => 50,
                 'hidden' => true,
-                'class' => ['form-row-wide', 'address-field'],
+                'class' => ['form-row-last', 'address-field'],
```

Comment:
maybe the css classes should be through woocomerce filter, looks really theme dependent

Reply:
it's generic class